### DISCUSSION TITLE
Local checkout breaks the shipping and checkout

### DISCUSSION THREAD

**Discussion by jwrhodes2008** [woocommerce/woocommerce#52202] [Checkout Flow]
The local checkout box being forced to appear and not hiding when the shipping area, location, etc. are not in the designated local tags breaks the cost calculation and fees from the buyer side. Meaning that a buyer can input their location and still press local pickup, then given a price that excludes shipping is ABLE TO PAY for a product without shipping included. This issue really should be addressed before you even have the code visible for use.

**Comment by juliaamosova** [comment]
Hi @jwrhodes2008! I am sorry to hear about the troubles you experiencing with shipping. There is not enough information for us to understand what could be going on. Can you please tell us exactly how your shipping is setup (screenshots will be helpful) so that we can help you further? Thank you!
